Libraries/Experimental/SwipeableRow/SwipeableListView.js

```diff
@@ -11,26 +11,49 @@
 'use strict';
 
 const ListView = require('ListView');
-const PropTypes = require('prop-types');
 const React = require('React');
 const SwipeableListViewDataSource = require('SwipeableListViewDataSource');
 const SwipeableRow = require('SwipeableRow');
 
-type DefaultProps = {
-  bounceFirstRowOnMount: boolean,
-  renderQuickActions: Function,
-};
+import type {Props as ListViewProps} from 'ListView';
+
+export type SwipeableListViewProps = $ReadOnly<{|
+  ...ListViewProps,
 
-type Props = {
+  /**
+   * To alert the user that swiping is possible, the first row can bounce
+   * on component mount.
+   */
   bounceFirstRowOnMount: boolean,
+  /**
+   * Use `SwipeableListView.getNewDataSource()` to get a data source to use,
+   * then use it just like you would a normal ListView data source
+   */
   dataSource: SwipeableListViewDataSource,
+  /**
+   * Maximum distance to open to after a swipe
+   */
   maxSwipeDistance:
     | number
     | ((rowData: any, sectionID: string, rowID: string) => number),
   onScroll?: ?Function,
-  renderRow: Function,
-  renderQuickActions: Function,
-};
+  /**
+   * Callback method to render the swipeable view
+   */
+  renderRow: (
+    rowData: any,
+    sectionID: string,
+    rowID: string,
+  ) => React.Element<any>,
```

Comment:
Should this be `React.Element` or `React.Node`? I'm not sure what this function can return. It probably is a React.Element. 👍

Reply:
I had a look around the codebase and for callbacks like this it seems to use `Element`